src/Accelerators/NNPA/Dialect/ZHigh/ZHighCombine.cpp

```diff
@@ -26,9 +26,24 @@ using namespace mlir;
 namespace {
 
 // Check if all values are produced by ZHighUnstickOp.
-bool areProducedByUnstickOp(ValueRange values, StringAttr layoutAttr) {
-  return llvm::all_of(values, [&layoutAttr](Value v) {
-    return isa<onnx_mlir::zhigh::ZHighUnstickOp>(v.getDefiningOp());
+bool areProducedByUnstickOp(ValueRange values) {
+  return llvm::all_of(values, [](Value v) {
+    using namespace onnx_mlir::zhigh;
+    // Block argument.
+    if (v.dyn_cast<BlockArgument>())
```

Comment:
[nit]: Can you use `isa` like at line 36 ?

Reply:
Done. Changed to use `isa`. I changed to use `isa` in another place also. Thank!